### ISSUE TITLE
Latest version on Nuget

### ISSUE THREAD

**Issue by ktheijs** [haplokuon/netDxf#365]
I am using netDxf through Nuget and I see that it is falling behind quite a bit. I have an issue where I do arcs/splines/... ToPolyline() but it returns a wrong result when the Normal of that entity is not (0,0,1). I believe this issue has been fixed in one of the latest commits.

Would it be possible to create a new release that includes all the latest commits and make it available on Nuget?

**Comment by GinoDerycke** [comment]
Yes, I was wondering too how to get the latest version on Nuget. I like the package but I would prefer to be able to download the latest stable version from Nuget.

**Comment by haplokuon** [comment]
I do not publish anything outside Github. The packages at Nuget are not mine, you will have to contact its actual owner.

**Comment by ktheijs** [comment]
Do you know who that is? On Nuget is says that you 'Daniel Carvajal' is the owner.

**Comment by haplokuon** [comment]
As far as I know there are three at nuget and none of them says that I am the owner. Go to the nuget website and search for them.

**Comment by JackClouseau** [comment]
I think 'thieum' is the owner of netDXF.
https://www.nuget.org/profiles/thieum

**Comment by droid1700** [comment]
I contacted thieum. A new version was posted today

**Comment by GinoDerycke** [comment]
Great, thx! Will the updates now be done periodically again on Nuget?

**Comment by dojo90** [comment]
> As far as I know there are three at nuget and none of them says that I am the owner. Go to the nuget website and search for them.

Why don't you publish it on nuget?

**Comment by DavidWishengrad** [comment]
Hey Dojo,

That question is for those who are putting his code elsewhere.  He wrote
the code, has others that contribute here, publishes here and treats people
well. That's more than enough.

Please consider using the search here. In your case with the word 'nuget'.

Also, has just recently noted that he is very busy.

Thank you for understanding. 👍

**Comment by dojo90** [comment]
@DavidWishengrad 
Well, it is of course very complicated to integrate a "nuget deploy job" into the build pipeline. *irony off*

But of course everyone has a different opinion on how to publish and present their library.

Thanks for your feedback anyway.

**Comment by haplokuon** [comment]
@dojo90 
Well, it is of course very complicated to download the netDxf project and press Build, _irony off_

If you are so interested in nuget why don't you do it? It is not like the MIT license does not allow it.
